recipe/meta.yaml

```diff
@@ -83,9 +85,11 @@ requirements:
     - vs2015_win-64                      # [win]
   host:
     - ncurses  # [not win]
-    - libiconv
+    - libiconv # [osx]
```

Comment:
Please try to keep the '# [....]' parts of line on same intend level

Reply:
Formatted the lines as suggested